src/App.tsx

```diff
@@ -1,10 +1,62 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
+
+import { getTodos } from './api/todos';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 11073;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [filterStatus, setFilterStatus] = useState(Status.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer) => {
+        setTodos(todosFromServer);
+      })
+      .catch((error: Error) => {
+        setHasError(true);
+        throw new Error(error.message);
```

Comment:
You are throwing error in catch block. This will result in the error being unhandled, leading to application crash. It's better to log the error message instead of throwing it.